File: Data Structures & Algorithms/palindromic-substrings/submission-2.cpp
class Solution {
public:
    bool isPalindromic(string s){
        int j = s.size();
        if(j<=1) return true ;
        int i=0;
        j--;
        while(i<j){
            if(s[i]!=s[j])
              return false ;
            i++;
            j--;
        }
        return true ;
    }
    int countSubstrings(string s) {
        int n= s.size();
        int count =0;
        if(n<=1) return n ;
        for(int i=0;i<n;i++){
            for(int j=i+1;j<=n ;j++){
                if(isPalindromic(s.substr(i,j-i)))
                   count++;
            }
        }
        return count ;
    }
};

File: Data Structures & Algorithms/anagram-groups/submission-2.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        R=[]
        i=1
        isdone=[]     
        for s in strs :
            if s not in isdone :
                E=[]
                for k in range(strs.count(s)) :
                    E.append(s)
                isdone.append(s)
                for j in range(i,len(strs)) :
                    if  "".join(sorted(s)) == "".join(sorted(strs[j])) and strs[j] not in isdone  :
                        E.append(strs[j])
                        isdone.append(strs[j])
                R.append(E)
                i+=1
        return R

